src/typing_extensions.py

```diff
@@ -2000,6 +2001,36 @@ def decorator(cls_or_fn):
         return decorator
 
 
+if hasattr(typing, "override"):
+    override = typing.override
+else:
+    _F = typing.TypeVar("_F", bound=typing.Callable[..., typing.Any])
+
+    def override(__arg: _F) -> _F:
```

Comment:
Just asking, my understanding was that types should not be added here and only provided via typeshed. I can't find a reference for it unfortunately. Did I miss something?

Reply:
It is true that type checkers will not look at this code. But types have value as documentation and perhaps some niche value for dynamic introspection.